AirCasting/SearchAndFollow/SearchLocation/SearchView.swift

```diff
@@ -9,9 +9,11 @@ struct SearchView: View {
     @StateObject var viewModel: SearchViewModel
     @Environment(\.presentationMode) var presentationMode
     @EnvironmentObject private var exploreSessionsButton: ExploreSessionsButton
+    @Binding var isSearchAndFollowFlowActive: Bool
```

Comment:
Since we are not even using this binding in this view, I think it would be good to remove it from this view and instead pass it ti the viewModel in the initialiser and then retrieve it from the view model when creating the SearchMapView